vision/genesis_principles.md

```diff
@@ -0,0 +1,14 @@
+# ₿trust Genesis Principles
+₿trust’s Genesis Principles are an immutable set of principles governing its operation and outlining its vision. They exist to guide the board and representatives of the trust, and to act as an accountability safeguard against bad actors.  
```

Comment:
```suggestion
₿trust’s Genesis Principles are an immutable set of principles governing its operation and outlining its vision. They exist to guide the board and representatives of the trust and will act as an accountability safeguard against bad actors.  
```

Reply:
Leaving as "to act" because "will" implies that they'll only do so at some (unspecified) point in the future?

Reply:
Yes, that's true, '_to act_' is more definite and binding here.